compose/ui/ui-test/build.gradle

```diff
@@ -107,6 +107,8 @@ if (AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 api(libs.kotlinCoroutinesCore)
                 api(libs.kotlinCoroutinesTest)
                 api(libs.kotlinStdlibCommon)
+
+                compileOnly("androidx.annotation:annotation:1.1.0")
```

Comment:
We intentionally [removed this annotation in the past](https://github.com/JetBrains/compose-multiplatform/issues/1186) from dependecnies of the desktop target, as it exists only in Google Maven, and works only for Android.

It is fine, if it is `compileOnly`, but why it is needed now?

Reply:
It was (and is) still present in `ui-test-junit4`.

It's needed for `androidx.annotation.VisibleForTesting`. Should I use `org.jetbrains.annotations.VisibleForTesting` instead?

Reply:
I see. I was the one that added it as compileOnly 😅:

<img width="1172" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/883267a3-47d0-407b-9b95-5f0e2156ab92">

Better to use `org.jetbrains.annotations` of course, but we should to do it in the Google repo. For now, simplier just to have `androidx.annotation`, it doesn't affect anything.